pmultiqc/modules/quantms/quantms.py

```
@@ -400,7 +400,15 @@ def __init__(self):
             self.xcorr_hist_range = {"start": 0, "end": 5, "step": 0.1}
             self.hyper_hist_range = {"start": 0, "end": 5, "step": 0.1}
             self.spec_evalue_hist_range = {"start": 0, "end": 20, "step": 0.4}
-            self.pep_hist_range = {"start": 0, "end": 1, "step": 0.02}
+            self.pep_hist_range = {
+                "start": 0,
+                "end": 1,
+                "low_thresh": 0.3,
+                "high_thresh": 0.7,
+                "end": 1,
```

Ruff (0.8.2)

Dictionary key literal `"end"` repeated

Remove repeated key literal `"end"`

(F601)